dev/tools/merge-pr.sh

```diff
@@ -91,8 +91,10 @@ fi
 REMOTE_URL=$(git remote get-url "$REMOTE" --all)
 if [ "$REMOTE_URL" != "${OFFICIAL_REMOTE_GIT_URL}" ] && \
    [ "$REMOTE_URL" != "${OFFICIAL_REMOTE_GIT_URL}.git" ] && \
-   [ "$REMOTE_URL" != "${OFFICIAL_REMOTE_HTTPS_URL}" ] && \
-   [ "$REMOTE_URL" != "${OFFICIAL_REMOTE_HTTPS_URL}.git" ]; then
+   [ "$REMOTE_URL" != "https://"*"@${OFFICIAL_REMOTE_HTTPS_URL}" ] && \
+   [ "$REMOTE_URL" != "https://${OFFICIAL_REMOTE_HTTPS_URL}" ] && \
+   [ "$REMOTE_URL" != "https://"*"@${OFFICIAL_REMOTE_HTTPS_URL.git}" ] && \
+   [ "$REMOTE_URL" != "https://${OFFICIAL_REMOTE_HTTPS_URL}.git" ]; then
```

Comment:
Not sure this works, as far I know you need `[[` and `]]` for `*` to mean wildcard.
Did you test it? Does it work for you?

Reply:
indeed I forgot the double [[ ]].